src/mmrelay/radio/base_backend.py

```
@@ -0,0 +1,65 @@
+
+    @abstractmethod
+    async def send_message(
+        self,
+        text: str,
+        channel: int | None = None,
+        destination_id: int | None = None,
+        reply_to_id: int | str | None = None,
+    ) -> Any:
+        """Send a message via the radio backend."""
+
+    def get_message_delay(self, config: dict[str, Any], default: float) -> float:
```

Ruff (0.14.14)

Unused method argument: `config`

(ARG002)